Sources/DocuCheckLib/Application/DatabaseOperations/UpdateDocumentTitles.swift

```diff
@@ -69,11 +69,21 @@ extension DocumentationDatabase {
         
         // Modify document
         document.remove(linesFrom: 0, count: line + 1)
+        
+        // Check if there is a template override
+        var layout = "page"
+        if let template = document.firstMetadata(withName: "TEMPLATE") {
+            if let params = template.parameters {
+                if (params.count == 1) {
+                    layout = params[0];
+                }
+            }
```

Comment:
I would add a warning in case that there's no parameter provided. For example:
```swift
         var layout = "page"
         if let template = document.firstMetadata(withName: "TEMPLATE") {
            if let layoutName = template.parameters?.first {
                layout = layoutName
            } else {
                Console.warning(document, template.beginLine, "Missing template name in TEMPLATE metadata tag.")
            }
         }
```